spec/unit/pops/parser/lexer2_spec.rb

```diff
@@ -91,12 +91,25 @@ def epp_tokens_scanned_from(s)
     "true"     => :BOOLEAN,
     "in"       => :IN,
     "unless"   => :UNLESS,
+    "private"  => :PRIVATE,
+    "type"     => :TYPE,
+    "attr"     => :ATTR,
```

Comment:
I don't understand this addition in context of PUP-4941. How was 'private', 'type', and 'attr' handled earlier?